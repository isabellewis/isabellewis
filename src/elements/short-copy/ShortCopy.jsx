import React from 'react';
import styled from 'styled-components';

export const ShortCopyP = styled.p`
	font-family: 'Archivo Narrow', sans-serif;
	font-size: 18px;
`;

export const ShortCopy = ({
	shortCopyText
}) => (
	<ShortCopyP className='short-copy-text'>
		{ shortCopyText }
	</ShortCopyP>
);
