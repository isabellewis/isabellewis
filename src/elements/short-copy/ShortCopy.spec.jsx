import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import { ShortCopy } from './ShortCopy';

describe('ShortCopy', () => {
	let component;

	beforeEach(() => {
		component = shallow(<ShortCopy title='ShortCopy test text' />);
	});

	it('should match the snapshot', () => {
		expect(component).toMatchSnapshot();
	});
});