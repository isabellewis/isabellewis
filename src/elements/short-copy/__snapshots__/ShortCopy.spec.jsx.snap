// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`elements/ShortCopy should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ShortCopy
    title="ShortCopy test text"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": undefined,
      "className": "short-copy-text",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": undefined,
        "className": "short-copy-text",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
